test/parallel/test-whatwg-url-searchparams-foreach.js

```diff
@@ -41,3 +41,10 @@ b = a.searchParams;
 for (i of b) {
   assert(false, 'should not be reached');
 }
+
+assert.throws(() => {
+  params.forEach.call(undefined);
+}, /^TypeError: Value of `this` is not a URLSearchParams$/);
+assert.throws(() => {
+  params.forEach();
+}, /^TypeError: The `callback` argument needs to be specified$/);
```

Comment:
This test seems to overlap with a series of similar, but more comprehensive tests in #10905. I'm fine with keeping it in this PR, but I'd prefer just dropping it here.